pylint/message/message_definition.py

```diff
@@ -18,6 +18,8 @@
 
 
 class MessageDefinition:
+    # TODO: remove disable and move all extra option to single parameter.
```

Comment:
```suggestion
```

Like I said in the issue. I don't think we should do this and just keep the extra disable.